src/gmmtypes.jl

```diff
@@ -64,8 +64,8 @@ types are supported: full covarariance and diagonal covariance.
     hist::Vector{History}
     "number of points used to train the GMM"
     nx::Int
-    function GMM(w::Vector{T}, μ::Matrix{T}, Σ::Union{DiagCov{T},FullCov{T}},
-                 hist::Vector, nx::Int)
```

Comment:
where is specified in the new syntax that Σ is a `DiagCov` or a `FullCov`?
Is the use of `AbstractMatrix` out of fashion?